test/integration/datapath/datapath_windows_test.go

```diff
@@ -123,14 +120,33 @@ func TestDatapathWin(t *testing.T) {
 
 		pods, err := k8sutils.GetPodsByNode(ctx, clientset, *podNamespace, podLabelSelector, node.Name)
 		if err != nil {
-			require.NoError(t, err)
+			t.Fatal(err)
 		}
 		if len(pods.Items) <= 1 {
-			require.NoError(t, errors.New("Less than 2 pods on node"))
+			t.Fatal("Less than 2 pods on node")
 		}
 	}
 	t.Log("Windows test environment ready")
+}
+
+func TestDatapathWin(t *testing.T) {
+	ctx := context.Background()
+
+	t.Log("Get REST config")
+	restConfig := k8sutils.MustGetRestConfig(t)
 
+	t.Log("Create Clientset")
+	clientset, _ := k8sutils.MustGetClientset()
+
+	setupWindowsEnvironment(t)
+	podLabelSelector := k8sutils.CreateLabelSelector(podLabelKey, podPrefix)
```

Comment:
Hi John, can you please rebased your branch, I think these codes are already there

Reply:
Just checked master, they are not.